src/h3lib/lib/algos.c

```diff
@@ -602,10 +603,11 @@ H3Error H3_EXPORT(gridDiskDistancesUnsafe)(H3Index origin, int k, H3Index *out,
             // the end of this ring.
             H3Error neighborResult = h3NeighborRotations(
                 origin, NEXT_RING_DIRECTION, &rotations, &origin);
-            if (neighborResult) {  // LCOV_EXCL_BR_LINE
+            if (neighborResult) {
                 // Should not be possible because `origin` would have to be a
                 // pentagon
-                return neighborResult;  // LCOV_EXCL_LINE
+                // TODO: Reachable via fuzzer
```

Comment:
For my info: Were these TODOs uncovered by trying to use the new macros here and triggering assertion errors?

Reply:
Yes, I generally tried to replace LCOV exclusions first with the macros and had to replace them with TODOs, since this PR makes it clear they are reachable.